drivers/unix/dir_access_unix.cpp

```diff
@@ -124,6 +124,26 @@ uint64_t DirAccessUnix::get_modified_time(String p_file) {
 	return 0;
 };
 
+// NEW FUNCTION
```

Comment:
These comments really are not necessary

Reply:
i know, it was only for mark the changes i have made. sorry for that